install.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright 2018 the Deno authors. All rights reserved. MIT license.
+# TODO(everyone): Keep this script simple and easily auditable.
+
+set -e
+
+deno_dir="$HOME/.deno/bin"
+deno_zip="${deno_dir}/deno.gz"
+deno_bin="${deno_dir}/deno"
+
+red="\033[31m"
+green="\033[32m"
+yellow="\033[33m"
+reset="\033[0m"
+
+if [ $# -eq 0 ]; then
+  # TODO: We could check the GitHub API for the latest release here.
+  # However, we need to parse the JSON response body. Perhaps
+  # this could be done with an existing installation of deno, jq,
+  # or node. And otherwise we ned to fall back to asking the user:
+  read -p "Which version do you want to install? " version
```

Comment:
This isn't helpful... The point of these scripts is to be a single command install.

Reply:
Yeah, that's kinda just a placeholder, until we decide on how to proceed with the "how to get the latest version?" problem.